packages/PyTrilinos/doc/Doxygen/doxy2swig.py.in

```diff
@@ -432,6 +433,30 @@ class Doxy2SWIG:
                 _tmp = textwrap.fill(i.strip(), break_long_words=False)
                 _tmp = self.lead_spc.sub(r'\1"\2', _tmp)
                 ret.extend([_tmp, '\n\n'])
+
+        # PytViper: look for pieces that have variadic template arguments
```

Comment:
It's been a while since I chatted with Bill Spotz -- this reminds me that SWIG needs to implement some part of a C++ compiler.

Reply:
Hmmmm... if Bill were to continue helping to maintain this, given his current position, he would have to do it under a pseudonym.

I am currently using SWIG 3.0.11.  3.0.12 has some issue I don't quite remember, making it unusable for PyTrilinos.  SWIG has graduated to version 4.0, which I tried briefly and it didn't work ... perhaps a problem for another day.

It looks like SWIG 3.0.11 parses the variadic template arguments correctly for the wrappers.  But that logic is not correctly implemented for the automatic generation of python docstrings.

Have a Merry Christmas!  Don't feel like you have to work on this over the holidays.  I just had some time to work on it during my family's drive to Colorado.

Reply:
Thanks for the well-wishes! :-D  I'm just procrastinating from other work ;-) .  I'm pretty sure Bill wouldn't have time to contribute to PyTrilinos these days, alas, even if he wanted to.  Are you planning to take over PyTrilinos development?  If so, I can see about adding you to the team so the `@`-mentions work.

Reply:
I *am* planning to take over PyTrilinos development.  A spare time kind of thing, which is better than nothing.  Being added to the team would be great, as long as it is low-profile.

Reply:
@pyt-viper wrote:
> I _am_ planning to take over PyTrilinos development. A spare time kind of thing, which is better than nothing.

Awesome! :-D  Could you please send an e-mail to my work account so I can get you signed up?  Thanks!